src/components/slider/slider.tsx

```diff
@@ -105,19 +107,31 @@ export class Slider {
     private mdcSlider: MDCSlider;
     private labelId: string;
     private helperTextId: string;
+    private observer: ResizeObserver;
 
     @State()
     private percentageClass: string;
 
     public constructor() {
+        const debounceTimeout = 200;
         this.inputHandler = this.inputHandler.bind(this);
         this.getContainerClassList = this.getContainerClassList.bind(this);
+        this.handleZoom = debounce(
+            this.handleZoom.bind(this),
```

Comment:
You shouldn't have to use `bind` when the original function is an arrow-function, as those bind `this` automatically. You don't have to change anything right now, but I wanted to comment on it for knowledge sharing, and also in case there is actually some reason this bind does actually need to be here 😊